juniper/src/integrations/chrono_tz.rs

```diff
@@ -0,0 +1,66 @@
+/*!
+
```

Comment:
I don't think we need this table. We only had it for chrono due to serde ambiguity .